app/pb_kits/playbook/pb_date_stacked/docs/_date_stacked_not_current_year.jsx

```diff
@@ -13,7 +13,7 @@ const DateStackedNotCurrentYear = () => {
       <br />
 
       <DateStacked
-          date={new Date('20 Mar 2018')}
+          date="2018-03-20"
```

Comment:
We want to be able to send either a `Date` object or a `String` here instead of just `String`.

Reply:
We do accept both. There was no examples with String, I just added one.

If you scroll up you can see a Date example
```
      <DateStacked
          date={new Date('20 Mar 2018')}
          size="sm"
      />

      <br />

      <DateStacked
          date="2018-03-20"
          size="md"
      />
```